pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -5614,3 +5614,34 @@ MULTILINESTRING ((0 0, 1 1), (1 1, 2.5 2.5), (1 3, 2.5 2.5), (2.5 2.5, 4 4), (4
 
 statement error geometry type is unsupported. Please pass a LineString or a MultiLineString
 SELECT ST_AsText(ST_Node(ST_GeomFromText('POLYGON((1 1, 1 3, 3 3, 3 1, 1 1))')));
+
+
+query T
+  SELECT
```

Comment:
nit: unindent
also use `ST_AsEWKT` and put an `SRID` on the polygon.

Reply:
Ok, updated